airflow/sensors/base.py

```diff
@@ -56,6 +56,25 @@ def _is_metadatabase_mysql() -> bool:
     return settings.engine.url.get_backend_name() == "mysql"
 
 
+class PokeReturnValue:
+    """
+    Sensors can optionally return an instance of the PokeReturnValue class in the poke method.
+    If an XCom value is supplied when the sensor is done, then the XCom value will be
+    pushed through the operator return value.
+    :param is_done: Set to true to indicate the sensor can stop poking.
+    :type is_done: bool
+    :param xcom_value: An optional XCOM value to be returned by the operator.
+    :type xcom_value: Optional[Any]
```

Comment:
```suggestion
    :param is_done: Set to true to indicate the sensor can stop poking.
    :param xcom_value: An optional XCOM value to be returned by the operator.
```

this will fix static check